### ISSUE TITLE
Make the contributors/repos/PRs items dynamic and hotlinks

### ISSUE THREAD

**Issue by rnelson0** [puppet-community/puppet-community.github.io#24]
The three items across the top appear static (https://github.com/puppet-community/puppet-community.github.io/blob/master/index.html#L32) and are not links to the appropriate sources.

**Comment by daenney** [comment]
Correct. Since pages are generated once this would need some client-side Javascript with the Github API to update those numbers.

**Comment by nibalizer** [comment]
Bump

**Comment by daenney** [comment]
Bump yourself :stuck_out_tongue:. I'm not comfortable enough with JS to do this.

**Comment by daenney** [comment]
Mostly fixed with #60.

**Comment by nibalizer** [comment]
This is the more correct answer anyways. Maybe we should have something in our docs/CoC/whatever encouraging people to publish their membership so our number gets bigger?

**Comment by daenney** [comment]
Ya, works for me.

**Comment by rnelson0** [comment]
So we just need the pull request data to be dynamic then?

**Comment by nibalizer** [comment]
Yes. See the commit message on #60. It isn't super easy, will require going over a list of repos, getting #s of PRs, then summing them.

**Comment by rnelson0** [comment]
Ha, I never noticed that, thought it ended up in the first convo message.

**Comment by bbriggs** [comment]
> This is the more correct answer anyways. Maybe we should have something in our docs/CoC/whatever encouraging people to publish their membership so our number gets bigger?

Yes.

**Comment by foxxx0** [comment]
I gave it a try, see: #91 :)

**Comment by bastelfreak** [comment]
also https://github.com/voxpupuli/voxpupuli.github.io/pull/92
